test/src/common/harnessHost.h

```diff
@@ -463,6 +463,7 @@ typedef struct HrnHostConfigUpdateParam
 {
     VAR_PARAM_HEADER;
     const Variant *archiveAsync;                                    // Update async archiving?
+    const Variant *spoolPathPostfix;                                // Add a postfix to the spool path?
```

Comment:
I think this comment wording would be more appropriate for a boolean variable like above, but we have a string variable.